lib/tasks/scrape.rake

```diff
@@ -18,7 +20,7 @@ namespace :scrape do
     subject_groups.each do |_code, group|
       puts "Updating group #{group["code"]}"
       subject_group = SubjectGroup.find_or_initialize_by(code: group["code"])
-      subject_group.name = capitalize_name(group["name"])
+      subject_group.name = capitalize_name(correct_typos(group["name"].downcase))
```

Comment:
This breaks subjects with roman numbers on their name so I'll have to change it

Reply:
Fixed in [f23d580](https://github.com/cedarcode/mi_carrera/pull/238/commits/f23d58043ecc8bef932e64963850242f6dda91ef) 🙂